src/main/java/com/faforever/client/game/GameTileController.java

```diff
@@ -97,16 +105,37 @@ public void setGame(Game game) {
         game.mapFolderNameProperty());
 
     JavaFxUtil.bind(gameMapLabel.textProperty(), mapNameBinding);
+
     numberOfPlayersLabel.textProperty().bind(createStringBinding(
         () -> i18n.get("game.players.format", game.getNumPlayers(), game.getMaxPlayers()),
         game.numPlayersProperty(),
         game.maxPlayersProperty()
     ));
-    mapImageView.imageProperty().bind(createObjectBinding(
-        () -> mapService.loadPreview(game.getMapFolderName(), PreviewSize.SMALL),
-        game.mapFolderNameProperty()
+
+    avgRatingLabel.textProperty().bind(createStringBinding(
+        () -> {
+          OptionalDouble avgRating = game.getTeams().values().stream()
+              .flatMap(Collection::stream)
+              .map(playerService::getPlayerForUsername)
+              .filter(Optional::isPresent)
+              .map(Optional::get)
+              .mapToInt(p -> RatingUtil.getRating(p.getGlobalRatingMean(), p.getGlobalRatingDeviation()))
+              .average();
+          if (avgRating.isPresent()) {
+            return String.format("%.0f", Math.round(avgRating.getAsDouble() / 100.0) * 100.0);
```

Comment:
define in message properties there are languages like chinese that might prefer other number formating